docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst

```diff
@@ -110,59 +110,85 @@ Rebuild The Yum Cache
 
    $ sudo yum makecache
 
-Install SIMP
--------------
-.. code-block:: bash
-
-   $ sudo yum install -y simp
+Install the SIMP Server
+-----------------------
 
-Modify Yum URLs
----------------
+1. Select the simp-adapter package appropriate for the version of Puppet
+   you will be using
 
-Set the following variables to repositories of your choosing in
-``/etc/puppetlabs/code/environments/production/hieradata/default.yaml``
+   * simp-adapter-foss:  Version appropriate for FOSS Puppet
+   * simp-adapter-pe:   Version appropriate for Puppet Enterprise
 
-.. code-block:: yaml
+2. Install the simp-adapter package
 
-   # Full URL to a YUM repo for Operating System packages
-   simp::yum::os_update_url: 'http://mirror.centos.org/centos/$releasever/os/$basearch/'
-   # Full URL to a YUM repo for SIMP packages
-   simp::yum::simp_update_url: 'https://packagecloud.io/simp-project/5_X/el/7/$basearch'
+.. code-block:: bash
 
-SIMP Config
------------
+   $ sudo yum install -y simp-adapter-foss
 
-Run simp config:
+3. Install the remaining SIMP packages
 
 .. code-block:: bash
 
-   $ simp config
+   $ sudo yum install -y simp
```

Comment:
This is a good place to mention simp-extra or whatever it is, as well as the fact that not all modules are installed by default.

They exist under the 'pupmod-*' RPM namespace.